src/sentry/replays/query.py

```diff
@@ -237,26 +253,63 @@ def _grouped_unique_scalar_value(
     )
 
 
-# Sort.
-
+# Filter
+
+replay_config = SearchConfig(
```

Comment:
nit: rename to `replay_search_config` to be a tiny bit clearer?

Reply:
Whoops forgot to push.